simphony_mayavi/sources/slim_cuds_source.py

```diff
@@ -0,0 +1,232 @@
+import itertools
+import numpy
+from traits.api import TraitError
+
+from simphony.core.cuba import CUBA
+from simphony.cuds import ABCMesh, ABCParticles, ABCLattice
+from simphony.io.h5_mesh import H5Mesh
+from simphony.testing.utils import dummy_cuba_value
+from simphony_mayavi.cuds.vtk_lattice import VTKLattice
+from simphony_mayavi.cuds.vtk_mesh import VTKMesh
+from simphony_mayavi.cuds.vtk_particles import VTKParticles
+from simphony_mayavi.sources.cuds_source import CUDSSource
+
+
+class SlimCUDSSource(CUDSSource):
+    """Overrides CUDSSource to provide a source optimized for lower memory
```

Comment:
Can I suggest to create a base `BaseCUDSSource` and having `CUDSSource`, `SlimCUDSSource` subclass from that?